ChangeLog

```diff
@@ -1,4 +1,4 @@
-2.1.21:
+2.2.0:
```

Comment:
you might want to specifically mention a change in the version number scheme in the changelog, no?